package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
While I don't mind this innocuous change, it is unrelated to the issue and best kept separate.